spec/1.1/index.bs

```diff
@@ -219,7 +219,7 @@ Ends presentation to the {{VRDisplay}} and fulfills the returned promise when fu
 Returns an array with the {{VRLayerInit}} currently being presented. MUST return an empty array if the {{VRDisplay}} is not currently presenting. If the {{VRDisplay}} is presenting MUST return an array containing the {{VRLayerInit}}s last passed to {{requestPresent()}}.
 
 <dfn method for="VRDisplay">submitFrame()</dfn>
-Captures the current state of the {{VRLayerInit}} currently being presented and displays it on the {{VRDisplay}}. It is assumed that the frame was rendered using the {{VRPose}} and matrices provided by the last call to {{getFrameData()}}. If {{getFrameData()}} was not called prior to calling {{submitFrame()}} the user agent MAY warn the user of potentially malformed visuals or prevent the frame from being shown at all.
+Captures the current state of the {{VRLayerInit}} currently being presented and displays it on the {{VRDisplay}}. It is assumed that the frame was rendered using the {{VRPose}} and matrices provided by the last call to {{getFrameData()}}. If {{getFrameData()}} was not called prior to calling {{submitFrame()}} the user agent MAY warn the user of potentially malformed visuals or prevent the frame from being shown at all. If any {{VRSource}} that was specified in the {{VRLayerInit}} array at {{requestPresent()}} time had no context, but a non-WebGL context was created subsequently after the fact, a NotSupportedError DOMException MUST be thrown, and the {{VRDisplay}} MUST NOT be updated with the contents of the layers specified by {{VRLayerInit}}.
```

Comment:
`displays it on` -> `displayed to`
`to calling {{submitFrame()}} the user agent` -> `to calling {{submitFrame()}}, the UA`